src/components/FormInputListContainer.tsx

```diff
@@ -9,26 +9,27 @@ import { RightIconButton } from "./RightIconButton";
 import { Box, Stack } from "@mui/material";
 import { ArrowForward } from "@mui/icons-material";
 import InputField from "./InputField";
-import { createContext, useState } from "react";
+import React, { createContext, useState } from "react";
```

Comment:
@bisratawoke It is better to import specific modules from react instead of importing the whole package.